src/Microsoft.Data.DataView/IDataView.cs

```diff
@@ -132,16 +132,18 @@ public abstract class DataViewRow : IDisposable
         public abstract ValueGetter<DataViewRowId> GetIdGetter();
 
         /// <summary>
-        /// Returns whether the given column is active in this row.
+        /// Returns whether the column with the given index, is active in this row.
         /// </summary>
-        public abstract bool IsColumnActive(int col);
+        public abstract bool IsColumnActive(int columnIndex);
```

Comment:
I thought we were changing these two methods to take in the column instead of the index?

`public abstract bool IsColumnActive(DataViewSchema.Column column);`

Reply:
I think the predicates were deemed hard to debug. Ideally we'd pass the column and not deal with the indexes but is that a V1 issue? @shauheen @glebuk @TomFinley

Reply:
The issue being closed here says:

> The use of integer indices here has sometimes led to confusion or even bugs.

Which seems to me that we shouldn't have integer indices in our public API. So if we ship V1 like this, we won't be able to remove/change this API. We can add a new `virtual` method that takes in the `Column`, but this current API will need to live forever.